src/style-spec/CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## 13.11.0-beta.1
+
+### ✨ Features and improvements
+* Add option to use a feature property as ID for feature state ([#8987](https://github.com/mapbox/mapbox-gl-js/pull/8987))
```

Comment:
Duplicated in the gl-js changelog

Reply:
I think they're intended to be duplicated, according to the process documentation.